### PR TITLE
(FB 151777) - Update title with relevant selection

### PR DESCRIPTION
**Pull request by benrudolph**
Updates HTML title tag in CloudCare with relevant selected Form,
Module, App.

### CODE REVIEW

File: corehq/apps/cloudcare/static/cloudcare/js/backbone/apps.js [hunk 1]
```diff
@@ -1123,7 +1129,13 @@ cloudCare.AppMainView = Backbone.View.extend({
         cloudCare.dispatch.on("module:deselected", function (module) {
             self.navigate("view/" + module.get("app_id"));
             self.clearModules();
+
+            self.updateTitle(self.app.get('name'))
```
Comment (dimagimon) [inline corehq/apps/cloudcare/static/cloudcare/js/backbone/apps.js]: Missing semicolon.

File: corehq/apps/cloudcare/static/cloudcare/js/backbone/apps.js [hunk 2]
```diff
@@ -1123,7 +1129,13 @@ cloudCare.AppMainView = Backbone.View.extend({
...
         });
+
+        cloudCare.dispatch.on('form:selected', function(form) {
+            self.updateTitle(form.get('name')[self.options.language]);
+        })
```
Comment (dimagimon) [inline corehq/apps/cloudcare/static/cloudcare/js/backbone/apps.js]: Missing semicolon.

File: corehq/apps/cloudcare/static/cloudcare/js/backbone/apps.js [hunk 3]
```diff
@@ -1139,6 +1151,10 @@ cloudCare.AppMainView = Backbone.View.extend({
             self.navigate("view/" + form.get("app_id") +
                                  "/" + form.get("module_index"));
             self.clearForms();
+
+            // After deselect form, update with selected module if it exists
+            var module = self.app.modules[self._selectedModule];
+            self.updateTitle(module ? module.get('name')[self.options.language] : null)
```
Comment (dimagimon) [inline corehq/apps/cloudcare/static/cloudcare/js/backbone/apps.js]: Missing semicolon.

**Comment by dimagimon** [comment]
:+1: No lint errors found.

**Comment by czue** [comment]
first PR! looks good.

**Comment by benrudolph** [comment]
:smile:

**Pull request merged**